CODEOWNERS

```diff
@@ -0,0 +1,71 @@
+# Scheduler
+master/internal/scheduler/    @aaron276h @rb-determined-ai
+
+# Provisioner
+master/internal/provisioner/  @shiyuann
+
+# Model registry
+# @eecsliu @katport
+
+# WebUI
+webui/                        @hkang1 @hamidzr
+
+# Experiment/trial actors
+master/internal/trial.go      @rb-determined-ai @dzhu @aaron276h
+master/internal/experiment.go @rb-determined-ai @dzhu @aaron276h
+
+# Actor infrastructure
+master/pkg/actor              @dzhu
+
+# D-train
+# @aaron276h @armandmcqueen
```

Comment:
@aaron276h @armandmcqueen is there a main set of files for this? I found this stuff distributed (heh) across many parts of the code

Reply:
Its a bit all over the place, https://github.com/determined-ai/horovod, a good amount of the harness and some scheduler stuff.

Reply:
do you have a set of candidate files/folders we should start with?

Reply:
https://github.com/horovod/horovod
https://github.com/determined-ai/determined/blob/master/harness/determined/horovod.py
https://github.com/determined-ai/determined/blob/master/harness/determined/ipc.py
https://github.com/determined-ai/determined/blob/master/harness/determined/layers/_worker_process.py
https://github.com/determined-ai/determined/blob/master/harness/determined/_trial_controller.py
https://github.com/determined-ai/determined/blob/master/harness/determined/exec/worker_process.py